lib/stdlib/doc/src/gen_event.xml

```diff
@@ -245,6 +245,16 @@ gen_event:stop             ----->  Module:terminate/2
                   Which term depends on the error.</p></item>
             </list>
           </item>
+          <item>
+              <p>If the calling process exits later, the event manager will
+              delete all handlers the calling process supervises (and call
+              <seemfa marker="#Module:terminate/2">
+              <c>Module:terminate({'EXIT', Pid, Reason}, State)</c></seemfa>),
+              then call
+              <seemfa market="#Module:handle_info/2">
```

Comment:
using _marker_ attribute instead of _market_ will fix the docs build error.
you can check "make docs" command yourself before pushing